src/main/java/org/jenkinsci/plugins/workflow/cps/GroovyClassLoaderWhitelist.java

```diff
@@ -85,9 +84,11 @@ private boolean checkJenkins26481(Object[] args, /* TODO Java 8: just take Execu
         if (permits(method.getDeclaringClass())) { // fine for source-defined methods to take closures
             return true;
         }
-        for (int i = 0; i < args.length; i++) {
-            if (args[i] instanceof CpsClosure && parameterTypes.length > i && parameterTypes[i] == Closure.class) {
-                throw new UnsupportedOperationException("Calling " + method + " on a CPS-transformed closure is not yet supported (JENKINS-26481); encapsulate in a @NonCPS method, or use Java-style loops");
+        if (false) { // TODO be more discriminating
```

Comment:
Suppressing the main change in #15 for the moment; #14 has something closer to what we need, though ideally we would check the actual method and determine if we have CPS coverage for _that_ method (and overload) now.